Sources/PostgresNIO/New/Connection State Machine/ConnectionStateMachine.swift

```diff
@@ -544,12 +544,12 @@ struct ConnectionStateMachine {
     }
     
     mutating func enqueue(task: PSQLTask) -> ConnectionAction {
-        let psqlErrror: PSQLError
+        let psqlError: PSQLError
```

Comment:
a bunch of types, not sure if i should move them to another PR or it's fine.

Reply:
Please create another PR for this.

Reply:
i'll remove the typo fixes then.